flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": [ "x86_64" ]
```

Comment:
this is disappointing.

Reply:
i am not really interested in installing another rust target rn

Reply:
what about aarch64 users then?